js/replay-embed.js

```diff
@@ -12,26 +12,26 @@ function requireScript(url) {
 	document.head.appendChild(scriptEl);
 }
 
-linkStyle('https://play.pokemonshowdown.com/style/font-awesome.css?');
-linkStyle('https://play.pokemonshowdown.com/style/battle.css?a7');
-linkStyle('https://play.pokemonshowdown.com/style/replay.css?a7');
-
-requireScript('https://play.pokemonshowdown.com/js/lib/jquery-1.11.0.min.js');
-requireScript('https://play.pokemonshowdown.com/js/lib/lodash.compat.js');
-requireScript('https://play.pokemonshowdown.com/js/lib/html-sanitizer-minified.js');
-requireScript('https://play.pokemonshowdown.com/js/lib/soundmanager2-nodebug-jsmin.js');
-requireScript('https://play.pokemonshowdown.com/js/config.js?a7');
-requireScript('https://play.pokemonshowdown.com/js/battledata.js?a7');
-requireScript('https://play.pokemonshowdown.com/data/pokedex-mini.js?a7');
-requireScript('https://play.pokemonshowdown.com/data/pokedex-mini-bw.js?a7');
-requireScript('https://play.pokemonshowdown.com/data/graphics.js?a7');
-requireScript('https://play.pokemonshowdown.com/data/pokedex.js?a7');
-requireScript('https://play.pokemonshowdown.com/data/moves.js?a7');
-requireScript('https://play.pokemonshowdown.com/data/abilities.js?a7');
-requireScript('https://play.pokemonshowdown.com/data/items.js?a7');
-requireScript('https://play.pokemonshowdown.com/data/teambuilder-tables.js?a7');
-requireScript('https://play.pokemonshowdown.com/js/battle-tooltips.js?a7');
-requireScript('https://play.pokemonshowdown.com/js/battle.js?a7');
+linkStyle('https://' + Config.routes.client + '/style/font-awesome.css?');
+linkStyle('https://' + Config.routes.client + '/style/battle.css?a7');
+linkStyle('https://' + Config.routes.client + '/style/replay.css?a7');
+
+requireScript('https://' + Config.routes.client + '/js/lib/jquery-1.11.0.min.js');
+requireScript('https://' + Config.routes.client + '/js/lib/lodash.compat.js');
+requireScript('https://' + Config.routes.client + '/js/lib/html-sanitizer-minified.js');
+requireScript('https://' + Config.routes.client + '/js/lib/soundmanager2-nodebug-jsmin.js');
+requireScript('https://' + Config.routes.client + '/js/config.js?a7');
```

Comment:
As you can see, it's not included until this specific line. And it won't actually exist until after this entire file has been run.